src/components/Article/ArticlesList.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react';
+import { getArticles, getArticleByID } from '../../api';
+import Article from './Article';
+
+function ArticlesList({ articles, setArticles }) {
+
+    const [isLoading, setIsLoading] = useState(true);
+
+    useEffect(()=> {
+        getArticles()
+        .then((fetchedArticles) => {
+            return fetchedArticles.map((article) => {
+                return getArticleByID(article.article_id)
+            })
```

Comment:
You don't need to get an article by id here. This is for when you click on an article and go to a separate page. 

You just need to get articles here and map over them like on line 27. Please remove this extra step from the useEffect and everything else should still work.